src/main/java/com/spikes2212/dashboard/Namespace.java

```diff
@@ -197,12 +197,13 @@ default void putBoolean(String key, boolean value) {
      */
     default PIDSettings addPIDNamespace(String name, PIDSettings initialPIDSettings) {
```

Comment:
add an overloading method that does not receive initial values and uses 0 as defualt